frontend/src/components/TicketCardClose.tsx

```diff
@@ -0,0 +1,232 @@
+import React from 'react';
+import {Card, CardContent, Typography, Box, useMediaQuery} from '@mui/material';
+import {styled} from '@mui/system';
+
+const Root = styled(Card)({
+  borderRadius: '24px',
+  overflow: 'visible',
+  backgroundColor: '#FFC107',
+  width: '441.25px',
+  height: '200px',
+  margin: '16px auto',
+  color: '#fff',
+  position: 'relative',
+});
+
+const Header = styled(Box)({
+  backgroundColor: '#FFC107',
+  padding: '8px 32px',
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  borderTopLeftRadius: '24px',
+  borderTopRightRadius: '24px',
+  color: '#fff',
+  fontWeight: 'bold',
+  position: 'absolute',
```

Comment:
Let's keep the `position: 'relative'`. This will fix the border-radius issue due to visible overflow.